forta-bots/Bridge-Invariant/src/constants.ts

```diff
@@ -1,25 +1,39 @@
-export interface NetworkData {
+import { Network } from "forta-agent";
+
+export const L2_MONITOR_HASH: string = "";
+
+interface L2Data {
   chainId: number;
   l1Escrow: string;
 }
 
-const L1_DAI: string = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
+export interface NetworkData {
+  DAI: string;
+  L2_DATA?: L2Data[];
+}
 
-const L2_DATA: NetworkData[] = [
-  {
-    chainId: 10, // optimism
-    l1Escrow: "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65",
-  },
-  {
-    chainId: 42161, // arbitrum
-    l1Escrow: "0xA10c7CE4b876998858b1a9E12b10092229539400",
+export type AgentConfig = Record<number, NetworkData>;
+
+export const CONFIG: AgentConfig = {
+  [Network.MAINNET]: {
+    DAI: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    L2_DATA: [
+      {
+        chainId: Network.OPTIMISM,
+        l1Escrow: "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65",
+      },
+      {
+        chainId: Network.ARBITRUM,
+        l1Escrow: "0xA10c7CE4b876998858b1a9E12b10092229539400",
+      },
```

Comment:
`L2_DATA` can be kept out of the network manager because it is only used on mainnet. This should be used for values that changing across networks, so as this values are only for Mainnet they can be regular variables passed directly to the L1 handler as parameters.

This is up to you. I am just sharing my thoughts on this.